.github/workflows/trivy-analysis.yml

```diff
@@ -0,0 +1,67 @@
+name: Trivy
+on:
+  workflow_dispatch:
+  # Uncomment this for testing purposes
+  #push:
+  #  branches: [ main ]
+  # Run the job twice a day  
+  schedule:
+    - cron: "0 0,12 * * *"
+
+jobs:
+  quarkus-dist:
+    name: Vulnerability scanner for Quarkus distribution images
+    runs-on: "ubuntu-18.04"
+    steps:
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
```

Comment:
Should we use SHA-pinning instead of `master`, or do we trust this GH Action? Or at least use some tag representing a specific version.